sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebViewHelper.java

```diff
@@ -38,12 +53,9 @@ final class HCaptchaWebViewHelper {
      * * loads custom html page to display challenge and/or checkbox
      */
     @SuppressLint({"SetJavaScriptEnabled", "AddJavascriptInterface"})
-    public void setup() {
-        final WebView webView = provider.getWebView();
-        if (webView == null) {
-            Log.w(HCaptcha.TAG, "WebView doesn't ready for setup");
-            return;
-        }
+    private void setupWebView() {
+        final HCaptchaJSInterface jsInterface = new HCaptchaJSInterface(config, provider, provider, provider, provider);
+        final HCaptchaDebugInfo debugInfo = new HCaptchaDebugInfo(context);
```

Comment:
Now we recreate our WebView bridges each time on setup, previously they were created once on the constructor. I don't think that this will have a significant performance impact (because JSInterface is really tiny, and DebugInfo cache the calculation, but for first captcha verification `HCaptchaDebugInfo` can take some time

Reply:
The `setupWebView` is called once from constructor. Has `private` modifier. What am I missing?

Reply:
You are right, I'm overlooked this)